### ISSUE TITLE
Filter dialogue box on column header click. 

### ISSUE THREAD

**Issue by harvetech** [olifolkerd/tabulator#3630]
Hey guys, sorry am posting here. Because no one replied on StackOverflow. I am new to tabulator. 
_`https://stackoverflow.com/questions/71284655/tabulator-header-filtering-via-drop-down-dialog-box`_

I will delete this issue as soon as someone notices my question. If such a functionality is not possible with the current tabulator I would highly suggest including it in future releases because a complete row is wasted even if you want to apply header filter to one column and it looks ugly. 

Thanks in advance.

**Comment by olifolkerd** [comment]
Hey @harvetech 

I'm afraid you need to have a bit more patience with Stack Overflow, developers answer things in their own time, and it has been less than a day since you first posted. You cant expect people to get back to you straight away.

I'm afraid i have a strict rule against questions on the issues list, otherwise it would become so crowded it would be unusable, so please don't try to boost responses to posts by linking from here in future.

As for the  feature request element of your issue, this has already been requested in issue #3568, please try searching the issue list first before posting.

Cheers

Oli :)

**Comment by harvetech** [comment]
@olifolkerd Sorry I have seen not so many views on my question and I saw in other questions you replied within a day and I was kind of having that hope too but ofc, I can't expect anyone to offer me help whenever I want so sorry for that. I actually searched for the issue already and commented on that 9 days ago. I also searched for your answers and tried to make some sense out of it if it's possible or not. 

Now everything is clear, I will delete this issue in a few hours once you have read my answer.
